apps/bandwagon/forms.py

```diff
@@ -143,6 +145,12 @@ def __init__(self, *args, **kw):
                 self.instance.type in amo.COLLECTION_SPECIAL_SLUGS):
             del self.fields['slug']
 
+    def clean(self):
+        if self.cleaned_data['your_name']:
+            raise forms.ValidationError(
+                'You\'ve been flagged as spam, sorry about that.')
```

Comment:
I used to think we had thoughts on strings re: `\` in our style guide but I checked and we don't :unamused: 

I always figured it was nicer to use double quotes for strings that would need escaping (eg `"It's"` over `'It\'s'`), but if other olympia code uses `\` then stick with this.